CoreFoundation/Preferences.subproj/CFPreferences.h

```diff
@@ -93,7 +93,7 @@ keysToFetch that are not present in the returned dictionary
 are not present in the domain.  If keysToFetch is NULL, all
 keys are fetched. */
 CF_EXPORT
-CFDictionaryRef CFPreferencesCopyMultiple(_Nullable CFArrayRef keysToFetch, CFStringRef applicationID, CFStringRef userName, CFStringRef hostName);
+_Nullable CFDictionaryRef CFPreferencesCopyMultiple(_Nullable CFArrayRef keysToFetch, CFStringRef applicationID, CFStringRef userName, CFStringRef hostName);
```

Comment:
Besides, as @spevans pointed out, it does return `NULL` in some condition. And the changes seems to work fine on both Linux & macOS. The only concern I have is the possibility of conflict if there will be any merging between the CF we used here & the CF in macOS.